charms/kfp-api/src/charm.py

```diff
@@ -252,6 +248,16 @@ def _generate_environment(self) -> dict:
             "ML_PIPELINE_VISUALIZATIONSERVER_SERVICE_HOST": viz_data["service-name"],
             "ML_PIPELINE_VISUALIZATIONSERVER_SERVICE_PORT": viz_data["service-port"],
             "CACHE_IMAGE": self.model.config["cache-image"],
+            # Configurations charmed-kubeflow adds to those of upstream
+            "ARCHIVE_CONFIG_LOG_FILE_NAME": self.model.config["log-archive-filename"],
+            "ARCHIVE_CONFIG_LOG_PATH_PREFIX": self.model.config["log-archive-prefix"],
+            # OBJECTSTORECONFIG_HOST and _PORT currently have no effect due to
```

Comment:
Could we change this comment to say that these variables will actually have an effect and that they will be used in the absence of other env variables like `MINIO_SERVICE_SERVICE_HOST` and a missing key in the `config.json`?